Use MacPorts txt2man (which is compatible with gawk 5) rather than the
older bundled version (which isn't).
--- Makefile.orig	2017-01-16 14:56:57.000000000 -0600
+++ Makefile	2021-03-17 01:13:02.000000000 -0500
@@ -23,8 +23,7 @@
 	echo "let pdfsandwich_version=\"$(VERSION)\";; (*automatically generated from file pdfsandwich_version*)" > $@
 
 $(MANUAL):	manual.txt
-	# you need gawk for this:
-	./txt2man -t PDFSANDWICH manual.txt | gzip -9 > $(MANUAL)
+	txt2man -t PDFSANDWICH manual.txt | gzip -9 > $(MANUAL)
 
 # Canceling pathological implicit rule:
 %: %.o
